ios/FluentUI/Core/Theme/Tokens/ControlTokenSet.swift

```diff
@@ -227,12 +227,12 @@ public enum ControlTokenValue {
 
     // MARK: - Helpers
 
-    private var fallbackColor: DynamicColor {
+    private var fallbackColor: UIColor {
 #if DEBUG
         // Use our global "Hot Pink" in debug builds, to help identify unintentional conversions.
-        return DynamicColor(light: ColorValue(0xE3008C))
+        return UIColor(colorValue: ColorValue(0xE3008C))
 #else
-        return DynamicColor(light: ColorValue(0x000000))
+        return UIColor(colorValue: ColorValue(0x000000))
```

Comment:
nit: could just be .black now?

Reply:
Gonna use `GlobalTokens` for both values